.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,91 @@
+name: Bug report
+description: Submit a new bug report.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## This issue tracker is only for technical issues related to Bitcoin Core.
+
+        * General bitcoin questions and/or support requests should use Bitcoin StackExchange at https://bitcoin.stackexchange.com.
+        * For reporting security issues, please read instructions at https://bitcoincore.org/en/contact/.
+        * If the node is "stuck" during sync or giving "block checksum mismatch" errors, please ensure your hardware is stable by running `memtest` and observe CPU temperature with a load-test tool such as `linpack` before creating an issue.
+
+        ----
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current behaviour
+      description: Tell us what went wrong
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected behaviour
+      description: Tell us what you expected to happen
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Tell us how to reproduce your bug. Please attach related screenshots if necessary.
+        * Run-time or compile-time configuration options
+        * Actions taken
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: |
+        Please copy and paste any relevant log output. This is normally the contents of a `debug.log` or `config.log` file.
+        This will be automatically formatted into code, so no need for backticks. If longer than 10,000 chars use a pastebin type site and just leave a link.
+      render: shell
```

Comment:
The docs say that users can upload files as long as `render` is not set. I think it would be better to let reporters upload their log files rather than requiring them to use a paste site.

Reply:
This could also help to avoid scenarios where thousands of lines of pasted logs make the issue harder to read.

Reply:
I made this change in 3fa1185dd. Seems like files can be added to the textarea with drag & drop